src/pages/Popup/components/AlarmUp.jsx

```diff
@@ -11,13 +11,15 @@ import Zoom from '@material-ui/core/Zoom';
 import PersonAddIcon from '@material-ui/icons/PersonAdd';
 import CloseIcon from '@material-ui/icons/Close';
 import DoneIcon from '@material-ui/icons/Done';
-
+import Timer from "./timer.jsx";
+import './timerstyle2.css';
```

Comment:
Don't create new css files for styling purpose
Add styles in the JSX file itself. Here is something you can use for your reference:
https://github.com/devlup-labs/meet-scheduler/blob/9e37ebe3578f29de49acde4297f4e24595fab4e9/src/pages/Popup/components/StudentForm.jsx#L15